swagger.yaml

```diff
@@ -4092,6 +4092,7 @@ definitions:
       - stop-installation
       - logs-gather
       - next-step-runner
+      - vsphere-disk-uuid-enabled-check
```

Comment:
maybe it will be better to call it: platform-validation-check and inside the step check if it's vsphere platform? I think it should be generic, @avishayt WDYT?

Reply:
The message, validation and UI behavior should be clear

Reply:
This should be a validation, not step type.  I agree that it should be generic, and add details to the message.